### ISSUE TITLE
inputs missing focused state

### ISSUE THREAD

**Issue by shimaasq** [square/maker#226]
### Bug description

Input fields are designed to have 1px border on hover and 2px border when focused. in the styleguide currently they only have hover state. I would like it to be updated with a focused state. if borders change the size of the element, shadows are also possible solution. 

the benefit of having hover and focused states is that they give the impression that the experience is fluid and buyer is in control of it, which create trust in the system. I understand that hover states are desktop centric but we still have big % of users on desktop that we want to serve good exp to.

Link to [design](https://www.figma.com/file/SHY7Z7PbcVssjIJVuJ3IwU/%F0%9F%8D%81-Maker?node-id=3710%3A5816)

![image](https://user-images.githubusercontent.com/75693425/154153787-d2198245-0d73-4d92-b9f8-7115154280ca.png)

### Reproduction

[n/a](https://square.github.io/maker/styleguide/pin-input/#/Input)

### Environment

```shell
https://square.github.io/maker/styleguide/pin-input/#/Input
```

### Addressed by

not urgent

### Can you contribute a fix?

- [ ] I’m interested in opening a pull request for this issue.

**Comment by pretzelhammer** [comment]
Hmmm, I investigated this a bit and I think I've narrowed it down [to this line as part of this release]( https://github.com/square/maker/compare/v4.0.1...v4.1.0#diff-43302d1852d8010ee012aab4dda2d01abb664e11f73617fe625f2ba112a958edR3)

Focus states were implemented using `:focus-within` which worked for all supported browsers at the time, up until Maker v4.1.0 when `>0.5%` was added to our supported browsers list which added browsers like UC Browser and Opera Mini which do not support `:focus-within` which basically silently broke focus states _for all browsers_ because postcss refused to process `:focus-within` anymore since it wasn't part of all our supported browsers anymore.

I think we can fix this by adding `not UCAndroid 12.12` and `not OperaMini all` to our browserslist.

**Comment by pretzelhammer** [comment]
input focus states are back: https://square.github.io/maker/styleguide/8.0.0/#/Input
